src/url.js

```diff
@@ -135,7 +135,7 @@ export function parseUrl(url, opt_nocache) {
  * @param {boolean=} opt_addToFront
  * @return {string}
  */
-function appendParamStringToUrl(url, paramString, opt_addToFront) {
+export function appendParamStringToUrl(url, paramString, opt_addToFront) {
```

Comment:
Used the first name. If you would prefer, I can copy this method into amp-analytics and keep it private.